rpm/deltarpm.spec

```diff
@@ -3,17 +3,17 @@
 
 Summary: Create deltas between rpms
 Name: deltarpm
-Version: 3.6.2
+Version: 3.6.3
 Release: 1
 License: BSD
 URL: https://github.com/rpm-software-management/deltarpm
 Source: %{name}-%{version}.tar.bz2
-Patch0: 0001-Do-not-build-with-zstd.-Set-prefix-to-usr.patch
 BuildRequires: bzip2-devel, xz-devel
 BuildRequires: pkgconfig(zlib)
 BuildRequires: pkgconfig(rpm)
 BuildRequires: pkgconfig(popt)
 BuildRequires: python3-devel
```

Comment:
Can you change this to also use pkgconfig?

Reply:
Changed